sdk/resourcemanager/Azure.ResourceManager/src/AzureStackProfile.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.ResourceManager
+{
+    /// <summary>
+    /// AzureStackProfile represents the information of an Azure Stack profile.
+    /// </summary>
+    public enum AzureStackProfile
+    {
+        /// <summary> The 2020-09-01-hybrid profile. </summary>
+        Profile20200901Hybrid
+    }
+
+#pragma warning disable SA1649 // File name should match first type name
```

Comment:
lets add this to its own class or our internal extensions class so we don't need this warn disable